Tasks/MSBuild/task.loc.json

```diff
@@ -108,11 +108,12 @@
       "label": "ms-resource:loc.input.label.msbuildVersion",
       "required": false,
       "groupName": "advanced",
-      "defaultValue": "14.0",
+      "defaultValue": "15.0",
```

Comment:
Making 15 an option is good but do we really want it to be the default?  It isn't released yet - not even RC.